Locked envs

Alex-Lewandowski opened the pull request:
MintPy Notebook updates:
- use `osl-mintpy` kernel (pins mintpy==1.5.1 due to prep_hyp3 issues)
- project to lat/lon in prep notebook

alexnaoki commented:
In this `SAR_Training/English/Master/Prepare_HyP3_InSAR_Stack_for_MintPy.ipynb` notebook, why revert to using `gdal.Warp()` since it has been removed from #209?

Alex-Lewandowski commented:
> In this `SAR_Training/English/Master/Prepare_HyP3_InSAR_Stack_for_MintPy.ipynb` notebook, why revert to using `gdal.Warp()` since it has been removed from #209?

Hi @alexnaoki We had to revert to using MintPy 1.5.1. Version 1.5.2 included an update that automatically projects from UTM to lat/lon, which is why we had removed the gdal.Warp() call. However, the reference point was still stored in the original UTM, which broke the GPS comparison section of `MintPy_Time_Series_From_Prepared_Data_Stack.ipynb` because we could no longer convert GPS site coordinates to XY coordinates using `mintpy.utils.utils.coordinate.geo2radar`.

We didn't have time to dig into a solution, so we just reverted to the previous version and resumed projecting to lat/lon in the data prep notebook. There was an open issue in MintPy on this, and checking just now, it looks like a PR was merged last week. We can take a look soon and move back to v1.5.2 if possible.
https://github.com/insarlab/MintPy/issues/1098
